chapters/packages.tex

```diff
@@ -227,12 +227,9 @@ \subsection{Directory Hierarchy Mapping}\label{mapping-a-package-class-hierarchy
 
 \subsection{Single File Mapping}\label{mapping-a-package-class-hierarchy-into-a-single-file-nonstructured-entity}\label{single-file-mapping}
 
-When mapping a package or class-hierarchy to a file (e.g., the file \filename{A.mo}), that file shall only define a single class \lstinline!A! with a name matching the name of the nonstructured entity.
-In a file hierarchy the files shall have the extension \filename{.mo}.
-
-A \filename{.mo} file defining more than one class cannot be part of the mapping
-to file-structure and it is an error if it is loaded from the
-\lstinline!MODELICAPATH!.
+When mapping a package or class-hierarchy to a file (e.g., the file \filename{A.mo}), the file content shall match \lstinline[language=grammar]!stored-definition! in the grammar.
```

Comment:
What is class-hierarchy if it is not a package? And also is it class-hierarchy or class hierarchy?

Reply:
This is a class hierarchy which not a package:
```
model A
  record R
  end R;
  R r;
end A;
```

This is the only occurrence of hyphened _class-hierarchy_, so it should clearly be spaced.  I'll fix.